src/pages/myAdmin/releaseNotes.tsx

```diff
@@ -0,0 +1,868 @@
+      </li>
+      <li>
+        For more information, refer to the{" "}
+        <a
+          href="https://docs.google.com/document/d/1hJj2CEMJlgbXHjF-it8Gey8b5_EnUDCqqhS8jRtxGyw/edit#heading=h.dzk3awl3nlud"
+          aria-label="Migrating to DIG from MyAdmin API/SDK guide"
+          target="_blank"
+          rel="noopener noreferrer">
+          Migrating to DIG from MyAdmin API/SDK guide
+        </a>
+        .
+      </li>
+      <li>
+        If you're a new Partner with a standalone telematics device, refer to the{" "}
+        <a
+          href="https://geotab.github.io/sdk/software/guides/custom-telematics-devices/"
+          aria-label="Using Custom Telematics Devices"
+          target="_blank"
+          rel="noopener noreferrer">
+          Using Custom Telematics Devices
+        </a>{" "}
+        section in the MyGeotab SDK.
+      </li>
+    </ul>
+  </div>
+);
+
+const updateSeptember122022: ReactNode = (
+  <div className="paragraph">
+    <h2>GetDeviceContracts</h2>
+    <ul>
+      <li>
+        <strong>GetDeviceContracts</strong> has been updated for Partners that closely integrate their billing with MyAdmin
+      </li>
+      <li>
+        The API now identifies the Rate Plan of a specific device in real time, as opposed to only the deivce mode
+      </li>
+    </ul>
+    <h2>Moving the MyAdmin SDK Site</h2>
+    <ul>
+      <li>
+        On September 17, 2022, the current MyAdmin SDK site will be deprecated, and the MyAdmin SDK will be merged with the MyGeotab SDK site to create a centralized point of information related to integrating with Geotab.
+      </li>
+    </ul>
+    <h2>End of Life — MyAdmin APIs for Third-party Device Data Ingestion</h2>
+    <ul>
+      <li>
+        On September 30, 2022, the MyAdmin APIs for third-party device data ingestion into MyGeotab will be permanently discontinued. After this date, MyGeotab will be unable to receive data from any devices that have not migrated to DIG.
+      </li>
+      <li>
+        For more information, refer to the{" "}
+        <a
+          href="https://docs.google.com/document/d/1hJj2CEMJlgbXHjF-it8Gey8b5_EnUDCqqhS8jRtxGyw/edit#heading=h.dzk3awl3nlud"
+          aria-label="Migrating to DIG from MyAdmin API/SDK guide"
+          target="_blank"
+          rel="noopener noreferrer">
+          Migrating to DIG from MyAdmin API/SDK guide
+        </a>
+        .
+      </li>
+      <li>
+        If you're a new Partner with a standalone telematics device, refer to the{" "}
+        <a
+          href="https://geotab.github.io/sdk/software/guides/custom-telematics-devices/"
+          aria-label="Using Custom Telematics Devices"
+          target="_blank"
+          rel="noopener noreferrer">
+          Using Custom Telematics Devices
+        </a>{" "}
```

Comment:
Need a TODO here so that we can validate the link eventually since this is a page in the SDK site.